spec/fixtures/slither/no-compile-truffle/contracts/bad-contract.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.8.4;
+
+contract C {
+    function f() internal returns (uint a) {
```

Comment:
Very minor comment feel free to ignore: Should we add a testcase for Ternary operator and assert that Slither fails? When we start supporting new version of slither(potentially with the fix), this testcase will fail and we will findout that the Ternary bug is fixed now

Reply:
I subscribed to the slither notifications so I'll get notified when there is a new release.